edgelet/edgelet-docker/tests/runtime.rs

```diff
@@ -328,11 +331,13 @@ fn container_create_succeeds() {
         vec![HostConfigPortBindings::new().with_host_port("8080".to_string())],
     );
     let memory: i64 = 3221225472;
+    let volumes = json!({"/test1": {}, "/test2": {}});
     let create_options = ContainerCreateBody::new()
         .with_host_config(
             HostConfig::new()
                 .with_port_bindings(port_bindings)
-                .with_memory(memory),
+                .with_memory(memory)
+                .with_binds(vec!["/a:/a".to_string()]),
```

Comment:
You are not asserting this anywhere looks like..

Reply:
Ah yes. I should remove this. This was orthogonal to my test.